velox/dwio/common/DataBuffer.h

```diff
@@ -177,6 +177,13 @@ class DataBuffer {
     size_ = (offset + items);
   }
 
+  void unsafeAppend(const T* src, uint64_t items) {
+    if (FOLLY_LIKELY(items > 0)) {
+      std::memcpy(data() + size_, src, sizeInBytes(items));
+      size_ = (size_ + items);
```

Comment:
```
nit: size_ += items;
```

Reply:
I realized there is a bug in the implementation of DataBuffer. `capacity` refers to the actual memory while `size` is the number of items. But at some places we compare capacity and size directly without `sizeInBytes(size)`. We probably always use a char data buffer and we don't notice this. I will fix in a separate PR.

Reply:
There is no bug. The code comments mislead me. I  will fix the comments.

Reply:
https://github.com/facebookincubator/velox/pull/6638